tools/docker/README.md

```diff
@@ -4,19 +4,22 @@ Asterinas development Docker images are provided to facilitate developing and te
 
 ## Building Docker Images
 
-To build a Docker image for Asterinas and test it on your local machine, navigate to the root directory of the Asterinas source code tree and execute the following command:
+Asterinas Docker image is based on an OSDK Docker image. To build a Docker image for Asterinas and test it on your local machine, navigate to the root directory of the Asterinas source code tree and execute the following command:
```

Comment:
Revise this paragraph similar to my comments about the OSDK Docker image documentation.